moveit_setup_assistant/templates/moveit_config_pkg_template/launch/moveit_rviz.launch

```diff
@@ -11,6 +11,7 @@
   <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" respawn="false"
 	args="$(arg command_args)" output="screen">
     <rosparam command="load" file="$(find [GENERATED_PACKAGE_NAME])/config/kinematics.yaml"/>
+    [KINEMATICS_PARAMETERS_FILE_NAMES_BLOCK]
```

Comment:
Actually, I believe the kinematics configuration should be properly loaded from the `/move_group` namespace, also in RViz. I would prefer to see this line (and the one above) removed from here unless you can confirm it is still needed.

Reply:
you're right, removed it